templates/docs/layouts/documentation.md

```diff
@@ -0,0 +1,86 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Documentation | Layouts
+---
+
+## Documentation layout
+
+<hr>
+
+### Structure
+
+The documentation page layout can be built using the Vanilla grid classes and common components. It consists of 3 horizontal areas that span the entire fixed width of the grid: header, content, footer.
+
+![Documentation layout structure](https://assets.ubuntu.com/v1/cdf09557-Documentation+layout%401x.svg)
+
+At the large breakpoint, the content area is further divided into an aside (3 columns) and a main content area (9 columns).
+
+At smaller breakpoints, the aside is moved offscreen and shown / hidden using a toggle.
+
+#### Header
+
+The header with the main navigation, built with the [navigation component](/docs/patterns/navigation#global-navigation).
+
+Style and contents of the documentation main navigation should be consistent with rest of the site.
+
+##### Search
+
+Documentation pages may have an optional search box in the main navigation.
+
+Alternatively search can be added in a full-width area under the top navigation, but above the aside and main content in a [strip component](/docs/patterns/strip) with grid row inside. Usually, it would be shallow light strip (`.p-strip--light is-shallow`), but the specific styling can be customised to match the site branding or other design requirements.
```

Comment:
```suggestion
Alternatively search can be added in a full-width area under the top navigation, but above the aside and main content in a [strip component](/docs/patterns/strip) with a grid row inside.
```

Reply:
Can we be more generic about the type of strip? I just advised @danielmutis to use a regular strip around a searchbox, because the content above (slanted suru) and below (logos) required a bit more separation.